exporters/otlp/otlpmetric/otlpmetricgrpc/internal/otest/collector.go

```diff
@@ -177,7 +177,7 @@ type HTTPResponseError struct {
 }
 
 func (e *HTTPResponseError) Error() string {
-	return fmt.Sprintf("%d: %s", e.Status, e.Err)
+	return e.Err.Error()
```

Comment:
We shouldn't lose the status code here. `%w` would allow unwrapping the error, and retrieving it back while retaining additional information.
The `%w` change is also happening in https://github.com/open-telemetry/opentelemetry-go/pull/5535

Reply:
This function is getting used while writing a response, `fmt.Fprintln(w, e.Error())` and as per earlier code it is sending status code in response while in actual collector response we only get error message not status code.

I completely agree on using %w when we are returning `error` but here we are returning a string.

Please correct if I am missing something here.

Reply:
Understood, I will make the change and update the other test cases accordingly, as there might be a few test cases which will break if status code with start showing in response.

Thanks, it makes sense.

Reply:
Addressed.